specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/disk.json

```diff
@@ -1344,6 +1344,157 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/diskAccesses/{diskAccessName}/privateEndpointConnections/{privateEndpointConnectionName}": {
+      "put": {
+        "tags": [
+          "DiskAccesses"
+        ],
+        "operationId": "PrivateEndpointConnections_Approve_Reject",
```

Comment:
What does this mean? I see it for other operations as well so curious.

Reply:
so this converts into the title of the operation _ becomes a - and a space would be added before every capitalized character.

So the title for this api would end up being Private Endpoint Connections - Approve - Reject

Ex: https://docs.microsoft.com/en-us/rest/api/compute/diskencryptionsets/listassociatedresources

Maybe i should rename this to DiskAccesses_UpdatePrivateEndpointConnection?

Reply:
Yes, there may be more properties to update in the future.

Reply:
updated the opids for all peConnection ops